src/Maestro/tests/Maestro.ScenarioTests/ObjectHelpers/SubscriptionStringBuilder.cs

```diff
@@ -0,0 +1,35 @@
+using System.Text;
+using Microsoft.DotNet.Darc;
+using Microsoft.DotNet.Maestro.Client.Models;
+
+namespace Maestro.ScenarioTests.ObjectHelpers
+{
+    /// <summary>
+    /// Contains the data for a subscription object and the logic for formatting into a darc return string, based on the code in Darc.GetSubscriptionsOperation
```

Comment:
Can we just use that code rather than re-implement it here?  Otherwise if we ever touch it, we need to come here and change it here too.

Reply:
I have a new version with this change - it would required refactoring the production function some though, so I definitely want to get a bunch of eyes on it to make sure I didn't inadvertently change something (based on my local testing there is no functional change, just a new function).